### PR TITLE
[CodeClean] remove unnecessary defines @open sesame 6/24 11:42

### PR DESCRIPTION
**Pull request by jaeyun-jung**
code clean, remove unnecessary predefinition to set pad template.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: ext/nnstreamer/tensor_source/tensor_src_tizensensor.c [hunk 1]
```diff
@@ -406,22 +395,15 @@ gst_tensor_src_tizensensor_class_init (GstTensorSrcTIZENSENSORClass * klass)
...
-#endif
+  /* pad template */
+  /** @todo Narrow down allowed tensors/tensor. */
+  pad_caps = gst_caps_from_string (GST_TENSOR_CAP_DEFAULT "; "
+      GST_TENSORS_CAP_WITH_NUM ("1"));
+  pad_template = gst_pad_template_new ("src", GST_PAD_SRC, GST_PAD_ALWAYS,
+      pad_caps);
+  gst_element_class_add_pad_template (gstelement_class, pad_template);
+  gst_caps_unref (pad_caps);
+
```
Comment (myungjoo) [inline ext/nnstreamer/tensor_source/tensor_src_tizensensor.c:406]: Do you support GST 1.0~1.7 with this code as well? (Tizen 4.0 uses GST 1.6.1)

Comment (jaeyun-jung) [reply ext/nnstreamer/tensor_source/tensor_src_tizensensor.c:406]: just removed `#if (GST_VERSION_MAJOR == 1) && (GST_VERSION_MINOR >= 8)`, other code is same.

File: gst/nnstreamer/include/tensor_typedef.h [hunk 2]
```diff
@@ -79,6 +79,14 @@
      * "dimensions = (string) dim1:dim2:dim3:dim4, dim1:dim2:dim3:dim4"
      */
 
+/**
+ * @brief Caps string for other/tensors with the number of tensors
+ */
+#define GST_TENSORS_CAP_WITH_NUM(num) \
+    "other/tensors, " \
+    "num_tensors = " num ", " \
```
Comment (myungjoo) [inline gst/nnstreamer/include/tensor_typedef.h:87]: is this safe? ("..." num "...")

Comment (jaeyun-jung) [reply gst/nnstreamer/include/tensor_typedef.h]: I updated the description for this macro, and changed default caps string for tensors using this. Please see the changes again.

File: gst/nnstreamer/tensor_repo/tensor_reposink.c [hunk 3]
```diff
@@ -129,22 +121,13 @@ gst_tensor_reposink_class_init (GstTensorRepoSinkClass * klass)
...
-        GST_TENSORS_CAP_DEFAULT);
-    pad_template = gst_pad_template_new ("sink", GST_PAD_SINK, GST_PAD_ALWAYS,
-        pad_caps);
-    gst_element_class_add_pad_template (element_class, pad_template);
-    gst_caps_unref (pad_caps);
-  }
-#else
-#error We support GStreamer 1.x only.
-#endif
+  /* pad template */
```
Comment (gichan-jang) [inline gst/nnstreamer/tensor_repo/tensor_reposink.c:124]: It seems to be used repeatedly. How about making it as util function?

File: ext/nnstreamer/tensor_source/tensor_src_tizensensor.c (hunk 1 above)
Comment (myungjoo) [reply ext/nnstreamer/tensor_source/tensor_src_tizensensor.c:406]: Tested at https://build.tizen.org/package/show/devel:Tizen:6.0:AI:GHM_TZN4/nnstreamer

File: gst/nnstreamer/tensor_repo/tensor_reposink.c (hunk 3 above)
Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_repo/tensor_reposink.c:124]: is it better to keep static GstStaticPadTemplate in each element and set from this?

Comment (gichan-jang) [reply gst/nnstreamer/tensor_repo/tensor_reposink.c:124]: Um, It seems better to define pads in each element. Thank you for your opinion.

**Pull request merged**